dist/index.js

```diff
@@ -7612,9 +7612,13 @@ async function get(url0, pageIdxs) {
       const headers = {
         'user-agent': 'setup-beam',
       }
-      if (process.env.GITHUB_TOKEN) {
-        headers.authorization = `Bearer ${process.env.GITHUB_TOKEN}`
+      const GithubToken =
+        getInput('github_token', false) || process.env.GITHUB_TOKEN
+
+      if (GithubToken) {
```

Comment:
We should probably do a check here to make sure Githubtoken isn't `""`, thoughts?